src/back/interfaces/room.py

```diff
@@ -13,3 +13,15 @@ def get_name(self: Self) -> str:
     def get_creation_datetime(self: Self) -> datetime:
         """Get room creation datetime."""
         raise NotImplementedError
+
+    def get_room_space(self: Self) -> int:
+        """Get room space."""
+        raise NotImplementedError
+
+    def get_users(self: Self) -> set[str]:
+        """Get users in a room."""
+        raise NotImplementedError
+
+    def join_user_to_room(self: Self, name: str) -> None:
+        """Join user to room."""
+        raise NotImplementedError
```

Comment:
Let's just make room simple dataclass. I feel like we are taking all these interfaces too far. It'll be much easier too work with.